test/unit/fixtures/normalizer/v1/sample-collection.js

```diff
@@ -0,0 +1,4360 @@
+                        key: 'client_id',
+                        value: 'abc123',
+                        type: 'text'
+                    },
+                    {
+                        key: 'client_secret',
+                        value: 'ssh-secret',
+                        type: 'text'
+                    }
+                ],
+                rawModeData: null,
+                descriptionFormat: null,
+                // eslint-disable-next-line max-len
+                description: 'Performing a simple `GET` request to this endpoint returns status code `401 Unauthorized` with `WWW-Authenticate` header containing information to successfully authenticate subsequent requests.\nThe `WWW-Authenticate` header must be processed to extract `realm` and `nonce` values to hash subsequent requests.\n\nWhen this request is executed within Postman, the script attached with this request does the hard work of extracting realm and nonce from the header and set it as [global variables](https://www.getpostman.com/docs/environments#global-variables) named `echo_digest_nonce` and `echo_digest_realm`.\nThese variables are re-used in subsequent request for seamless integration of the two requests.',
+                headers: '',
+                method: 'GET',
+                pathVariables: {},
+                url: 'https://echo.getpostman.com/digest-auth',
+                preRequestScript: '',
+                // eslint-disable-next-line max-len
+                tests: 'tests[\'response code is 401\'] = responseCode.code === 401;\ntests[\'response has WWW-Authenticate header\'] = (postman.getResponseHeader(\'WWW-Authenticate\'));\n\nvar authenticateHeader = postman.getResponseHeader(\'WWW-Authenticate\'),\n    realmStart = authenticateHeader.indexOf(\'\'\',authenticateHeader.indexOf(\'realm\')) + 1 ,\n    realmEnd = authenticateHeader.indexOf(\'\'\',realmStart),\n    realm = authenticateHeader.slice(realmStart,realmEnd),\n    nonceStart = authenticateHeader.indexOf(\'\'\',authenticateHeader.indexOf(\'nonce\')) + 1,\n    nonceEnd = authenticateHeader.indexOf(\'\'\',nonceStart),\n    nonce = authenticateHeader.slice(nonceStart,nonceEnd);\n    \npostman.setGlobalVariable(\'echo_digest_realm\', realm);\npostman.setGlobalVariable(\'echo_digest_nonce\', nonce);',
+                currentHelper: 'normal',
+                helperAttributes: {},
+                collectionId: '03cf74df-32de-af8b-7db8-855b51b05e50'
+            },
+            {
+                folder: '3448365f-6534-c34d-14ed-01915810374e',
+                id: 'b213d463-8587-8470-acdc-92f7dfb0660e',
+                name: 'Get UTF8 Encoded Response',
+                dataMode: 'params',
+                data: [],
+                rawModeData: null,
+                descriptionFormat: null,
+                // eslint-disable-next-line max-len
+                description: 'If a response of an endpoint requires to send data beyond the basic English / ASCII character set, the `charset` par
```

Comment:
Normalized collection requests should not have `currentHelper` and `helperAttributes` if auth type is `normal`